lib/WP_Auth0_Nonce_Handler.php

```diff
@@ -39,6 +39,14 @@ class WP_Auth0_Nonce_Handler {
 	 */
```

Comment:
The pattern for the additions to this file was pulled from the PHP SDK:

https://github.com/auth0/auth0-PHP/pull/395/files#diff-edb92fb2369f39801bab7466d95cbc1c